code/game/machinery/iv_drip.dm

```diff
@@ -112,12 +112,14 @@
 	if(src.attached && src.beaker)
 		// Give blood
 		if(mode)
-			if(src.beaker.volume > 0)
-				var/transfer_amount = REAGENTS_METABOLISM
+			if(beaker.volume > 0)
 				if(beaker.reagents.reagent_list.len == 1 && beaker.reagents.has_reagent(BLOOD))
 					// speed up transfer if the container has ONLY blood
-					transfer_amount = 4
-				src.beaker.reagents.trans_to(src.attached, transfer_amount)
+					beaker.reagents.trans_to(attached, 4)
```

Comment:
lets be real here this + below transfer should really be faster, it's way better to just syringe blood into people